src/crypto-providers/util.ts

```diff
@@ -199,37 +199,26 @@ const filterSigningFiles = (
   }
 }
 
-const hwSigningFileToPubKeyHash = (signingFile: HwSigningData): Uint8Array => (
-  cardanoSerialization.PublicKey.from_bytes(
-    splitXPubKeyCborHex(signingFile.cborXPubKeyHex).pubKey,
-  ).hash().to_bytes()
+const hwSigningFileToPubKey = (signingFile: HwSigningData): Buffer => (
+  splitXPubKeyCborHex(signingFile.cborXPubKeyHex).pubKey
 )
 
-const findPathForKeyHash = (
-  keyHash: Buffer, signingFiles: HwSigningData[],
-): BIP32Path | undefined => {
-  const signingFile = signingFiles.find((file) => keyHash.equals(hwSigningFileToPubKeyHash(file)))
-  return signingFile?.path
+const hwSigningFileToPubKeyHash = (signingFile: HwSigningData): Uint8Array => {
+  const pubKey = hwSigningFileToPubKey(signingFile)
+  return cardanoSerialization.PublicKey.from_bytes(pubKey).hash().to_bytes()
 }
 
-const findSigningPathForKeyHash = (
```

Comment:
These two functions seem to behave exactly the same. I discussed with Jakub why he added `findPathForKeyHash` while keeping `findSigningPathForKeyHash` (the original one) as well, but he said he doesn't remember and recommended keeping only one of them. I chose the one which uses `cardanoSerialization` since we consider it generally more up-to-date than `cardanoCrypto` if I understand correctly (I remember something like this from previous conversations, please correct me if this is wrong).

Reply:
That sounds good.